packages/react-core/src/components/Button/examples/ButtonWithCount.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { BadgeCountObject, Button } from '@patternfly/react-core';
+
+export const ButtonWithCount: React.FunctionComponent = () => {
+  const badgeCountObjectNotRead: BadgeCountObject = {
+    isRead: false,
+    count: 7,
+    className: 'custom-badge-unread'
+  };
+
+  const badgeCountObjectRead: BadgeCountObject = {
+    isRead: true,
+    count: 10,
+    className: 'custom-badge-read'
+  };
+
+  return (
+    <React.Fragment>
+      <p>unread:</p>
```

Comment:
Nit - the read/unread paragraph text should probably be sentence case?

```suggestion
      <p>Unread:</p>
```